docs/integrations/working-with-custom-scripts/custom-script-samples.md

```diff
@@ -38,74 +38,85 @@ Please be aware that this custom script affects any step that comes after. There
+    puts "Coverage is #{line_coverage} and below minimum coverage percentage given #{min_coverage}. \nExiting."
+    exit (1)
+else
+    puts "Coverage is above the threshold. It is clear."
+end
 
-# Subject line for email
-EMAIL_SUBJECT="Appcircle Build"
+rescue StandardError => e
+  puts "An error occurred: #{e.message}"
+end
+```
 
-# Send a corresponding email for successful builds.
+:::info
 
-echo "Build completed successfully, sending email notification."
-echo -e "Your build in Appcircle is completed successfully." | mail -s "Build completed with success." ${RCP_ADDRESS} -c ${CC_ADDRESS}
-echo "Email notification sent."
-```
+Please feel free to edit the following variables according to your own data:
+- `test_result_file_path`: The file path of the test result file from which to retrieve the covered percentage value.
+- `min_coverage`: The minimum percentage required for the pipeline to continue without breaking.
 
-###
+:::
 
-### Send Slack notification when a build is complete
+### Deploying Apps to Firebase App Distribution
 
-Appcircle supports a wide range of options for sending notifications to Slack, but you can also send any custom messages to Slack with custom scripts.
+Appcircle Testing Distribution provides an integrated and automated enterprise-grade solution for distributing apps to the testers, but if you want to use other solutions for app distribution, you can do so with custom scripts. You can use the following script below to deploy apps to Firebase App Distribution automatically from the Appcircle Build module.
 
-If you have a Slack webhook created, you can send a message to your Slack channel with a single line of Bash script.
+- The binary to be deployed can be obtained with the related environment variable. `AC_EXPORT_DIR` and the binary path.
+- `FIREBASE_TOKEN` must be obtained through a local console. Please follow the [instructions here](https://firebase.google.com/docs/cli#cli-ci-systems) to set up the Firebase CLI locally and then you can request a token with the `firebase login:ci `command.
+- `FIREBASE_APP_ID` can be obtained from the Firebase Dashboard under the settings screen:
 
-You can refer to Slack's webhook documentation here: :link: [**Creating Slack webhooks**](https://api.slack.com/tutorials/slack-apps-hello-world)
+<Screenshot url='https://cdn.appcircle.io/docs/assets/image (133).png' />
 
+The sample script is as follows:
 
 ```bash
 #!/usr/bin/env bash
 #
-# Send a Slack notification when a build is completed in Appcircle.
+# Deploy a binary to Firebase App Distribution
 #
+curl -sL firebase.tools | bash
 
-curl -X POST -H 'Content-type: application/json' --data '{"text":"Appcircle build is completed successfully!"}' SLACK_WEBHOOK_URL
+firebase appdistribution:distribute $AC_EXPORT_DIR/Runner.ipa --app $FIREBASE_APP_ID --release-notes "Release Notes..." --token $FIREBASE_TOKEN --groups "testers"
 ```
+
+You may also use our Firebase App Distribution Component for this process.
```

Comment:
We can give a link to make a directions to the related component documentation page.